src/_rules/color.js

```diff
@@ -12,6 +12,7 @@ export const textColors = [
 //  ['text-inherit', { 'color': 'inherit' }], // This class currently sets "text-align: inherit;" in align.js
   ['text-transparent', { 'color': 'transparent' }],
   ['text-current', { 'color': 'currentColor' }],
+  [/^text-\[(.+)]/, ([, p]) => ({ 'color': p.startsWith('var') ? p : `var(${p})` })],
```

Comment:
Have you also discussed accepting arbitrary values for text size? We have support for arbitrary values for line-height in [/_rules/typography.js](https://github.com/warp-ds/drive/blob/next/src/_rules/typography.js#L12):
```
[/^leading-\[(.+)(rem|px)?\]/, ([, value, unit], context) => ({ 'line-height': resolveArbitraryValues(value, unit, context) })]
```

If it was considered to target classes with `/^text-\[(\d+)(rem|px)?\]$` then the rule like `/^text-\[(.+)]/` might take over.